opensearch/src/main/java/org/opensearch/sql/opensearch/executor/OpenSearchExecutionEngine.java

```diff
@@ -57,9 +75,9 @@ public void explain(PhysicalPlan plan, ResponseListener<ExplainResponse> listene
         Explain openSearchExplain = new Explain() {
           @Override
           public ExplainResponseNode visitTableScan(TableScanOperator node, Object context) {
-            return explain(node, context, explainNode -> {
-              explainNode.setDescription(ImmutableMap.of("request", node.explain()));
-            });
+            return explain(node, context, explainNode ->
+              explainNode.setDescription(ImmutableMap.of("request", node.explain()))
+            );
```

Comment:
Do we need these changes?

Reply:
Regarding the switch from 
`            return explain(node, context, explainNode -> {
              explainNode.setDescription(ImmutableMap.of("request", node.explain()));
            });
           `
           to `            return explain(node, context, explainNode ->
              explainNode.setDescription(ImmutableMap.of("request", node.explain()))
            );`
            It was giving me a warning about "Warning:(60, 58) Statement lambda can be replaced with expression lambda".

Reply:
That is an IDEA warning/suggestion. We can ignore them unless they in code we're working on.